examples/pom.xml

```diff
@@ -21,6 +21,10 @@
         <failOnMissingWebXml>false</failOnMissingWebXml>
         <version.microprofile.bom>${version.wildfly}</version.microprofile.bom>
         <version.server.bom>${version.wildfly}</version.server.bom>
+        <version.org.wildfly.arquillian>3.0.0.Final</version.org.wildfly.arquillian>
+        <version.org.jboss.arquillian.junit>1.7.0.Alpha4</version.org.jboss.arquillian.junit>
```

Comment:
We should use 1.6.0.Final here since that's what the WildFly Arquillian adapter uses.

Reply:
Thank-you, fixed.